asm/unk_0202EB30.s

```diff
@@ -178,8 +178,8 @@ _0202EC68: .word 0x000005D8
 _0202EC6C: .word 0x00000678
 	thumb_func_end sub_0202EC28
 
-	thumb_func_start GetMigratedPokemonByIndex
-GetMigratedPokemonByIndex: ; 0x0202EC70
+	thumb_func_start MigratedPokemon_ToPokemon
```

Comment:
tbh I think this might be better as MigratedPokemon_ConvertToPokemon

Reply:
> tbh I think this might be better as MigratedPokemon_ConvertToPokemon

Applied 👍